### PR TITLE
Content state utils

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/common/web/statics/schemas/content-state.schema.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
             "title": "text",
             "default": "",
             "examples": ["test"],
-            "pattern": "^(.*)$"
+            "pattern": "^([^]*)$"
```
Comment (lxgreen) [inline packages/common/web/statics/schemas/content-state.schema.json]: WTF?

File: packages/common/web/src/Utils/data-schema-validator.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import contentStateSchema from '../../statics/content-state.schema.json';
+import contentStateSchema from '../../statics/schemas/content-state.schema.json';
 
-export const validate = (data, schema) => {
+export const validate = (data, schema, alert = true) => {
```
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/common/web/src/Utils/data-schema-validator.js]: I thought that returning true or false was the desired behaviour, I'll delete it.

File: packages/common/web/statics/schemas/content-state.schema.json (hunk 1 above)
Comment (talevy17) [reply packages/common/web/statics/schemas/content-state.schema.json]: The previous pattern didn't accept line breaks in code block for some reason (e.g \n)
I tried a few, this was the only one that passed my tests

Comment (lxgreen) [reply packages/common/web/statics/schemas/content-state.schema.json]: still this is weird way to accept any symbol. the `[^]` means "everything except ...nothing". please use `^(.|\n)*$` instead

Comment (talevy17) [reply packages/common/web/statics/schemas/content-state.schema.json]: Done.

**Pull request merged**